treeherder/perf/models.py

```diff
@@ -53,6 +53,9 @@ class PerformanceSignature(models.Model):
     # generous max_length permits up to 8 verbose option names
     extra_options = models.CharField(max_length=360, blank=True)
 
+    # TODO: reduce length to minimum value
+    measurement_unit = models.CharField(max_length=50, null=True)
```

Comment:
Please also add another *separate* TODO for making this nonnullable, once we demand all PERHERDER_DATA dumps to provide the `unit` field.

Reply:
Actually, this was done before amending this commit. I changed it because I wanted to adapt unit tests. But, in order to adapt them to the new changes, I needed to see which test case fails. So I made this. I fixed the tests and I will do your proposed change (basically, reverting to the nonnullable situation). Thanks.

Reply:
Ok